src/editor/controller/editor_controller.cpp

```diff
@@ -1284,6 +1284,17 @@ void editor_controller::mouse_motion(int x, int y, const bool /*browse*/,
 	gui().highlight_hex(hex_clicked);
 }
 
+void editor_controller::touch_motion(int x, int y, const bool browse, bool update, map_location new_loc)
+{
+	(void) x;
+	(void) y;
+	(void) browse;
+	(void) update;
+	(void) new_loc;
```

Comment:
For these you should just comment out the parameter names, like so: `int /*x*/`, etc.